### PR TITLE
Introducing `commonwealth`

### PR DESCRIPTION
**Pull request by rafaellehmkuhl**
The Commonwealth of Services, generally known simply as the Commonwealth, is a technological association of Companion services to share common knowledge.

### CODE REVIEW

File: core/services/commonwealth/setup.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    name="commonwealth",
+    version="0.1.0",
+    description="Companion library to share common code.",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    license="MIT",
+    packages=setuptools.find_packages(),
+    python_requires=">=3.9",
+    install_requires=[
+        "starlette == 0.13.6",
```
Comment (patrickelectric) [inline core/services/commonwealth/setup.py:16]: missing loguru

File: core/services/install-services.sh [hunk 2]
```diff
@@ -36,6 +36,9 @@ cd /home/pi/services/versionchooser/ && python3 setup.py install
 # Ping service:
 cd /home/pi/services/ping/ && python3 setup.py install
 
+# Commonwealth library:
+cd /home/pi/services/commonwealth/ && python3 setup.py install
```
Comment (patrickelectric) [inline core/services/install-services.sh:40]: why this is running after all services if it appears as base dependence for all of them ?

File: core/services/commonwealth/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Commonwealth of Services
```
Comment (patrickelectric) [inline core/services/commonwealth/README.md:1]: What is the reason to have an empty .gitignore ?
We can add it when necessary.

**Review by patrickelectric — CHANGES_REQUESTED** [review]
nice cleanup

File: core/services/commonwealth/README.md (hunk 3 above)
Comment (rafaellehmkuhl) [reply core/services/commonwealth/README.md:1]: No specific reason besides o.c.d :smile:

File: core/services/install-services.sh (hunk 2 above)
Comment (rafaellehmkuhl) [reply core/services/install-services.sh:40]: Moving up.

File: core/services/commonwealth/setup.py (hunk 1 above)
Comment (rafaellehmkuhl) [reply core/services/commonwealth/setup.py:16]: I had put it there but apparently I removed on the last rebase. Fixing.

**Comment by Williangalvani** [comment]
it looks like a library to me. I would place it in core/libs/commonwealth.

**Review by patrickelectric — CHANGES_REQUESTED** [review]
wifi and ardupilot_manager are missing in setup.py the dependency for `commonwealth == 0.1`

**Comment by rafaellehmkuhl** [comment]
> it looks like a library to me. I would place it in core/libs/commonwealth.

@patrickelectric can we settle with /core/libs/commonwealth so?

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**